src/app/components/ConnectorDrawer/ConnectorDrawer.tsx

```diff
@@ -183,24 +142,17 @@ export const ConnectorDrawerPanelContent: FunctionComponent<ConnectorDrawerPanel
       getNamespace({
         accessToken: getToken,
         connectorsApiBasePath: connectorsApiBasePath,
-        namespaceId: namespaceId,
+        namespaceId: connectorData.namespace_id,
       })(getNamespaceData, onError);
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [namespaceId]);
 
-    useEffect(() => {
       setKIData('');
       getKafkaInstanceById({
         accessToken: getToken,
         kafkaManagementBasePath: kafkaManagementApiBasePath,
-        KafkaInstanceId: kafkaId,
+        KafkaInstanceId: connectorData.kafka.id,
       })(getKIData, onKIError);
       // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [kafkaId]);
-
-    const selectActiveKey = (_: MouseEvent, eventKey: string | number) => {
-      setActiveTabKey(eventKey);
-    };
+    }, [connectorData]);
 
     const { connector } = useConnector(
```

Comment:
What's this connector vs `connectorData` that's being passed in?

Reply:
replied above.